tests/roles/glance_adoption/tasks/main.yaml

```diff
@@ -40,7 +40,6 @@
         enabled: true
         template:
           databaseInstance: openstack
-          containerImage: {{ container_registry|default("quay.io") }}/{{ container_namespace|default("podified-antelope-centos9") }}/openstack-glance-api:{{ container_tag | default("current-podified") }}
```

Comment:
ack we have https://github.com/openstack-k8s-operators/data-plane-adoption/blob/677d1913ebb583b649178d457413653d34966e92/tests/roles/backend_services/templates/container_overrides.j2#L18-L23